receiver/pulsarreceiver/Makefile

```diff
@@ -1 +1,6 @@
 include ../../Makefile.Common
+
+# Remove "-race" from the default set of test arguments.
+# exporter/pulsarexporter tests are failing with the -race check.
+# See https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/12518
```

Comment:
Please crate another issue for the receiver

Reply:
Still an incorrect issue mentioned

Reply:
OK, I forgot update the issue number